spec/integration/list_spec.rb

```diff
@@ -854,9 +855,9 @@ class ::Property
 
             item.position = nil  #  NOTE:: Creates a messed up original list.
             item.user = @u2
-            item.save
+            item.list_scope.should_not == item.original_list_scope
 
-            item.list_scope.should != item.original_list_scope
+            item.save
```

Comment:
Save clears original attributes. I had to change this a few times. This is my only concern.